src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -63,7 +63,7 @@ internal ImmutableArray<TypeWithAnnotations> TypeArgumentsWithDefinitionUseSiteD
 
             foreach (var typeArgument in result)
             {
-                typeArgument.Type.OriginalDefinition.AddUseSiteDiagnostics(ref useSiteDiagnostics);
```

Comment:
>Type [](start = 29, length = 4)

Why not simply access DefaultType here?

Reply:
It would be good to avoid repeating `type.Default.OriginalDefinition.AddUseSiteDiagnostics()`. Moved the helper method here.

---
In reply to: [493955260](https://github.com/dotnet/roslyn/pull/47968#discussion_r493955260) [](ancestors = 493955260)